example-kotlin/src/main/java/com/attentive/example_kotlin/ExampleKotlinApp.kt

```diff
@@ -0,0 +1,34 @@
+package com.attentive.example_kotlin
+
+import android.app.Application
+import com.attentive.androidsdk.AttentiveConfig
+import com.attentive.androidsdk.AttentiveEventTracker
+import com.attentive.androidsdk.UserIdentifiers
+
+class ExampleKotlinApp : Application() {
+    lateinit var attentiveConfig: AttentiveConfig
+
+    companion object {
+        fun buildUserIdentifiers(): UserIdentifiers {
+            // Add all the identifiers that you have for the current user
+            return UserIdentifiers.Builder()
+                .withPhone("+15556667777")
+                .withEmail("some_email@gmailfake.com")
+                .withKlaviyoId("userKlaviyoId")
+                .withShopifyId("userShopifyId")
+                .withClientUserId("userClientUserId")
+                .withCustomIdentifiers(mapOf(Pair("customIdentifierKey", "customIdentifierValue")))
+                .build()
+        }
+    }
+
+    override fun onCreate() {
+        super.onCreate()
+        attentiveConfig = AttentiveConfig("games", AttentiveConfig.Mode.PRODUCTION, this)
```

Comment:
can we use the "YOUR_ATTENTIVE_DOMAIN" placeholder here?

Reply:
yep, done